src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -998,6 +998,18 @@ void regclass_graph_Model(py::module m) {
                 :type path: str
              )");
 
+    model.def(
+        "_get_raw_address",
+        [](ov::Model& self) {
+            return reinterpret_cast<uint64_t>(&self);
+        },
+        R"(
+        Returns raw address of the Model object.
+
+        :return: a raw address of the Model object.
+        :rtype: int
```

Comment:
```suggestion
        Returns a raw address of the Model object from C++.
        
        Use this function to compare underlying C++ addresses instead of using `__eq__` in Python.

        :return: a raw address of the Model object.
        :rtype: int
```